source/frontend/src/features/mapSideBar/compensation/list/CompensationListContainer.tsx

```diff
@@ -39,8 +39,15 @@ export const CompensationListContainer: React.FC<ICompensationListContainerProps
   const location = useLocation();
   const { setModalContent, setDisplayModal } = useModalContext();
 
+  const [fileCompensationResults, setfileCompensationResults] =
+    useState<ApiGen_Concepts_CompensationRequisition[]>(null);
+  const [subFilesCompensations, setSubFilesCompensations] =
+    useState<ApiGen_Concepts_CompensationRequisition[]>(null);
+  const [resultsLoading, setResultsLoading] = useState<boolean>(false);
+
   const {
     updateAcquisitionFile: { execute: putAcquisitionFile },
+    getAcquisitionSubFiles: { execute: fetchSubFiles },
```

Comment:
Note, these repository methods come with their own loading boolean flags, you don't need to create your own loading state tracker.

Reply:
I know. But I am making a lot requests in batch and I did not want an intermittent Boolean flag which I don't know how it would reflect in the component.

Reply:
I know. But I am making a lot requests in batch and I did not want an intermittent Boolean flag which I don't know how it would reflect in the component.